### PR TITLE
Realign dialog buttons.

### PR DESCRIPTION
**Pull request by jqlee85**
## Summary

Addresses issue[ #1436](https://github.com/google/site-kit-wp/issues/1436)

Updated CSS for .mdc-dialog__actions

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/sass/vendor/_mdc-dialog.scss [hunk 1]
```diff
@@ -24,7 +24,8 @@
 	}
 
 	.mdc-dialog__actions {
-		justify-content: flex-start;
+		justify-content: space-between;
+		margin: 2em 0;
```
Comment (tofumatt) [inline assets/sass/vendor/_mdc-dialog.scss:28]: The space _below_ the buttons here looks odd to me. I know the IB said `2em` but it seems like quite a lot, especially below the button. What does `1em` look like?

https://github.com/google/site-kit-wp/pull/1458/files#diff-abeb2930ae311ff937d08ecc500f6f3c to me is a bit too much whitespace.

Comment (jqlee85) [reply assets/sass/vendor/_mdc-dialog.scss:28]: Yeah @tofumatt 1em looks better to me, should I go ahead and commit?
modal-dialog-1em

Comment (tofumatt) [reply assets/sass/vendor/_mdc-dialog.scss:28]: Looks good, :shipit:

Comment (jqlee85) [reply assets/sass/vendor/_mdc-dialog.scss]: Great, updated and pushed. 👍

**Pull request merged**